server/controllers/cohort.js

```diff
@@ -293,7 +294,7 @@ const deleteCohortById = (req, resp) => {
       const { ownerIds: owners, name } = cohort;
       const data = { cohortId, owners };
 
-      socketActions.deleteCohort(socketInstance, allCohortsViewClients)(data);
+      socketActions.deleteCohort(io.getInstance(), allCohortsViewClients)(data);
```

Comment:
Please extract instance first to reduce diff and complexity.

```suggestion
      const socketInstance = io.getInstance();
      socketActions.deleteCohort(socketInstance, allCohortsViewClients)(data);
```